src/query/frontend/opencypher/grammar/MemgraphCypher.g4

```diff
@@ -259,7 +259,13 @@ updateClause : set
 
 foreach :  FOREACH '(' variable IN expression '|' updateClause+  ')' ;
 
-indexHints: USING INDEX indexHint ( ',' indexHint )* ;
+usingStatements: USING usingStatement ;
```

Comment:
I think this is a wrong naming for the left value since you already have the * operator in the cypherQuery:

so it needs to be singular
usingStatement: USING usingStatementItem
usingStatementItem: hopsLimit | indexHints ;